### ISSUE TITLE
trees follow my view

### ISSUE THREAD

**Issue by Tackietacktack** [tsunamayo/Starship-EVO#5467]
### Short Description of the issue:

in the video see you of the leftside plants which follow my view and bugged grabbed things in my right hand but that is a other bug.
i have test a surival start and produce many components and a little bit mining.
i had mining on trees but that´s was befor restart the game.
sorry i don´t have more for reproducing.

new observation:
After restarting the trees are still there.

### Environment
- Starship EVO version: 24w05a
- Graphics Card: NVIDIA GeForce RTX 2060

### Additional information:

[Player.log](https://github.com/tsunamayo/Starship-EVO/files/14229016/Player.log)

https://github.com/tsunamayo/Starship-EVO/assets/97230091/6aa3ccc6-6143-4fb3-8974-bb4a727d6aed

**Comment by Gallt** [comment]
I had rocks do this on planet too once.

**Comment by tsunamayo** [comment]
@Tackietacktack Hi, if you get this after a reload that is interesting, could you send me your character save? Thanks

**Comment by Tackietacktack** [comment]
yes after reload and yesterday i have it too, but now i don´t have follower.  can it be, that you repair it with #5480 (-Tree are not dropping wood)?

**Comment by tsunamayo** [comment]
@Tackietacktack Yes I need it all, I dont know if the upload will work though.

**Comment by asanagisae** [comment]
I get the same issue sometimes. However, the irregular trees and rocks will disappear after restarting the save.

Reproduction steps: Run around a lot in generally one direction? I picked up more stray items with my ship than on foot. These trees I got by flying towards the peak of the mountain (in the back), then turning around and flying back to base, approximately straight but not exactly.

https://github.com/tsunamayo/Starship-EVO/assets/10069469/90c9a1e3-7150-40dc-b343-a2d415723bf9
